darts/tests/models/forecasting/test_global_forecasting_models.py

```diff
@@ -815,3 +821,122 @@ def test_residuals(self):
 
         res = model.residuals(ts)
         assert len(res) == 38 - (24 + 12)
+
+    @pytest.mark.parametrize(
+        "params",
+        product(
+            [
+                (
+                    {"series": sine_3_ts + 3},
+                    {"target": Scaler()},
+                ),
+                (
+                    {"series": sine_3_ts + 3, "past_covariates": sine_1_ts + 3},
+                    {"past": Scaler(scaler=MinMaxScaler())},
```

Comment:
That is the default already

```suggestion
                    {"past": Scaler()},
```